californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseStatus.java

```diff
@@ -126,6 +126,15 @@ public int getCurrentNum() {
 	}
 
 	/**
+	 * Returns true if this is the first block in the sequence.
+	 *
+	 * @return true if first block, false otherwise.
+	 */
+	public boolean isFirst() {
+		return currentNum == 0;
+	}
+
```

Comment:
I like this :-) We are doing a lot of such comparisons outside of the objects that should know best about their internal state ...